R/AllGenerics.R

```diff
@@ -799,10 +816,24 @@ setGeneric("featureDefinitions<-", function(object, value)
 #' spectrum **per chromatographic peak** will be returned (hence multiple
 #' spectra per feature).
 #'
-#' The ID of each chromatographic peak (i.e. its row name in `chromPeaks`)
-#' and each feature (i.e., its row name in `featureDefinitions`) are
-#' available in the returned [Spectra()] with spectra variables `"peak_id"`
-#' and `"feature_id"`, respectively.
+#' The information from `featureDefinitions` for each feature can be included
+#' in the returned [Spectra()] object using the `addColumnsFeatures` parameter.
```

Comment:
also here - maybe use `featureColumns` instead? or turn it round and use `addFeatureColumns`?

Reply:
`featureColumns` is a bit less obvious than `chromPeakColumns` I think but the equivalent would be `featureDefinitionsColumns` and that is way too long. let's go with `featureColumns`, the description is clear enough, user will understand.